<!DOCTYPE html
	PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"> 
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en   "> 
 
  <head> 
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /> 
	<title> 
	  NetResponse: functional network analysis
	</title> 
	<link href="http://r-forge.r-project.org/themes/rforge/styles/estilo1.css" rel="stylesheet" type="text/css" /> 
  </head> 
 
<body> 
 
<style type="text/css"> 
 
body{
	text-align: center;
	max-width:950px;
	min-width:950px;
	margin-left:auto;
	margin-right:auto;
}
 
a img{
	border:none;
}
 
div#wrapper{
	text-align: left;
}
 
div.h2{
	margin:19px 0px 19px 0px;
	font-size:16pt;
	font-weight:bold;
	background-color:#135CAE;
	color:white;
	padding:10px;
}
 
div.h3{
	margin:19px 0px 19px 0px;
	font-size:12pt;
	font-weight:bold;
}
 
h3, div.h3{
	background-color:#dddddd;
	padding:2px;
}
 
div#abstract{
	font-size: 10pt;
}
 
div#support{
	font-size: 10pt;
}
 
img{
	vertical-align: middle;
}
 
.comment{
	color: #770000;
}
 
div.menu th{
	padding:8px;
	border-top:1px solid gray;
	border-bottom:1px solid gray;
}
 
th.link{
	padding:8px;
	border-top:1px solid gray;
	border-bottom:1px solid gray;
	border-left:1px dotted gray;
	background-color:#dddddd;
}
div.menu th.link a{
	color:#111199;
}
 
div.menu td{
	border-bottom:1px solid gray;
	border-left:1px solid gray;
}
 
div.menu th:first-child {
	border-left:1px solid gray;
	background-color:#dddddd;
} 
 
div.menu th.link:hover, div.menu th.link:hover a{
	background-color:#135CAE;
	color: white;
}
 
div.menu th.selected, div.menu th.selected a{
	background-color:#ffffff;
	color: black;
	border-bottom:none;
}
 
.textbf{
	font-weight: bold;
}
 
 
</style> 
 
<div id="wrapper"> 
<div class="h2">NetResponse: probabilistic algorithms for functional network analysis
</div> 
<table width="100%" cellpadding="0" cellspacing="0"> 
<tr> 
<td valign="top"> 
Leo Lahti*, Olli-Pekka Huovilainen, Ant&oacute;nio Gusm&atilde;o and Juuso Parkkinen
<br /> 
<small> 
(*) University of Helsinki, Finland; <a href="http://www.iki.fi/Leo.Lahti">corresponding author</a></p> 
</small> 

<p></p><p></p> 


NetResponse provides probabilistic tools for functional analysis of
interaction networks, and provides a global map of network activation
patterns across versatile contexts. The implementations are based on
probabilistic models and variational learning to deal rigorously with
inherent uncertainty in real-life data, and to allow incorporation of
prior information in the analysis. Currently available tools:

<ul>

<li> <b>NetResponse<a href="http://bioinformatics.oxfordjournals.org/content/early/2010/09/02/bioinformatics.btq500.full.pdf?ijkey=N1zzjtUBRng3wCM&keytype=ref">&sup1;</a></b>
to detect and characterize local, context-specific activation patterns
and to construct global functional maps of large interaction networks
by utilizing functional information of the network based on
non-parametric analysis with variational Dirichlet process Gaussian
mixture models&sup2;.
<p></p>
<li> <b>Interaction Component Model (ICMg&sup3;)</b>: to discover
functional network modules, or <i>communities</i>, taking into account
the uncertainty in network structure. Module discovery can be
supervised by functional information of the network.
<p></p>
<li> <b>Visualization: </b>Preliminary tools are available to
visualize netresponse output. Install
the <a href="netresponse.visualization.tar.gz">netresponse
visualization package</a> and check the package vignette (in package
source, inst/doc/) for examples.

</ul>

Applicability of the models has been demonstrated by case studies in
computational biology
(<a href="http://bioinformatics.oxfordjournals.org/content/early/2010/09/02/bioinformatics.btq500.abstract.html?ijkey=N1zzjtUBRng3wCM&keytype=ref">Lahti
et al., 2010</a>, Parkkinen and Kaski, 2010), where the algorithms
have been used to investigate the structure and context-specific
transcriptional activity of genome-scale interaction networks in human
body.

<p></p> The techniques are implemented
in <a href="http://www.r-project.org">R</a>, an open source
environment for statistical computing. For installation instructions,
see the <a target="_new"
href="http://www.bioconductor.org/help/bioc-views/devel/bioc/html/netresponse.html">project
page at BioConductor</a>.  For further documentation, see
the <a href="http://www.bioconductor.org/packages/2.7/bioc/vignettes/netresponse/inst/doc/netresponse.pdf">package
vignette</a>. In
addition, <a href="http://www.cis.hut.fi/projects/mi/software/NetResponse/">Matlab
source code</a> for the NetResponse algorithm&sup1; is available.


</td> 
<td>
<img src="methodVertical7.jpg" width="170" alt="" />
</td> 
</tr></table> 

<!--<h3>Installation</h3>
Each module has to be installed independently. For installation
instructions and description of the modules, see the
package <a href="../pkg/inst/doc/dmt.pdf">vignette</a>. Note
that this is a development version.
-->


<h3>Licensing terms</h3>

Licensed under an open source software license (GNU GPL >=2).  Your
feedback and contributions are welcome.

<!--<h3>Acknowledgements</h3> 
<div id="support"> 
The project is a joint effort by several
people: <a href="http://www.iki.fi/Leo.Lahti">Leo Lahti</a>, 
Olli-Pekka Huovilainen, and Ant&oacute;nio Gusm&atilde;o.
</div> 
-->

<h3>References</h3> 
<div id="support"> </div>

<ol>

<li> Leo Lahti et al. (2010). <a href="http://bioinformatics.oxfordjournals.org/content/early/2010/09/02/bioinformatics.btq500.full.pdf?ijkey=N1zzjtUBRng3wCM&keytype=ref"><b>Global
modeling of transcriptional responses in interaction
networks</b></a>. <i>Bioinformatics</i>. (<a href="http://bioinformatics.oxfordjournals.org/content/early/2010/09/02/bioinformatics.btq500.abstract.html?ijkey=N1zzjtUBRng3wCM&keytype=ref">abstract</a>; <a href="http://bioinformatics.oxfordjournals.org/content/early/2010/09/02/bioinformatics.btq500.full.pdf?ijkey=N1zzjtUBRng3wCM&keytype=ref">advanced online-access (pdf)</a>; <a href="http://www.cis.hut.fi/lmlahti/publications/Lahti10bioinf-preprint.pdf">preprint with supplementary figures (pdf)</a>
<p></p>

<li> Kenichi Kurihara et al. (2007). <b>Accelerated variational
  Dirichlet process mixtures</b>. In B. Sch&ouml;lkopf, J. Platt, and
  T. Hoffman, eds., <i>Advances in Neural Information Processing
  Systems 19</i>, 761-768. MIT Press, Cambridge, MA.</li>
<p></p>
<li> Juuso Parkkinen and Samuel Kaski. <b>Searching for functional 
gene modules with interaction component models.</b> 
<em>BMC Systems Biology</em> 2010, 4:4. 

</ol>





</div> 
</body> 
</html> 
